lib/order.rb

```diff
@@ -0,0 +1,21 @@
+class Order
+
+  attr_reader :order
+
+  def initialize
+    @order = {}
+  end
+
+  def add item, number, menu
+    line= menu.select_food(item.to_s)
+    line.each{|k,v| k=k, v[1]=number}
+    @order.merge!(line)
```

Comment:
You have a reader and `merge!` is an 'in-place' editor, so you should use the reader throughout here.